src/sentry/integrations/slack/message_builder/notifications/base.py

```diff
@@ -65,11 +65,7 @@ def build(self) -> SlackAttachment | SlackBlock:
 
         actions_block = []
         for action in actions:
-            if (
-                action.label in ["Turn on personal notifications", "Approve", "Reject"]
-                or action.url
-            ):
```

Comment:
was this intentional?

Reply:
yes this was, we don't need this because this logic covers every single action created